virtualnetworks.go

```diff
@@ -15,13 +15,15 @@ type ProjectVirtualNetworkService interface {
 }
 
 type VirtualNetwork struct {
-	ID           string  `json:"id"`
-	Description  string  `json:"description,omitempty"`
-	VXLAN        int     `json:"vxlan,omitempty"`
-	FacilityCode string  `json:"facility_code,omitempty"`
-	CreatedAt    string  `json:"created_at,omitempty"`
-	Href         string  `json:"href"`
-	Project      Project `json:"assigned_to"`
+	ID           string    `json:"id"`
+	Description  string    `json:"description,omitempty"`
+	VXLAN        int       `json:"vxlan,omitempty"`
+	FacilityCode string    `json:"facility_code,omitempty"`
+	CreatedAt    string    `json:"created_at,omitempty"`
+	Href         string    `json:"href"`
+	Project      *Project  `json:"assigned_to,omitempty"`
+	Facility     *Facility `json:"facility,omitempty"`
+	Instances    []*Device `json:"instances,omitempty"`
```

Comment:
Maybe "Devices" instead of "Instances"? Unless there'a plan to change the terminology in Equinix Metal.

Reply:
I expressed the same concern. https://github.com/packethost/packngo/issues/241#issuecomment-776097615

This has been brought to the attention of the API team too.

There is only one other place that the API calls these Instances instead of Devices and that is in the Swagger spec of the SpotMarketRequests.  In SpotMarketRequest responses, the actual response includes `devices` instead of the documented `instances` so that leaves only `VirtualNetworks`.

I'm a little torn between representing the API faithfully and providing a consistent packngo UX.

I think I'd rather lean on a faithful API representation. By doing so, generated documentation will match this client and potentially generated clients.

Reply:
OK